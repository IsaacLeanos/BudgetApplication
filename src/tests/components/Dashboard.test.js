import React from 'react'
import{shallow}from'enzyme'
import ExpenseDashboardPage from '../../components/Dashboard'

test('render dashboard component',()=>{
    let wrapper=shallow(<ExpenseDashboardPage/>)
    expect(wrapper).toMatchSnapshot()
})