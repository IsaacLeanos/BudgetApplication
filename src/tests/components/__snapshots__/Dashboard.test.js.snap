// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render dashboard component 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseList) />
</div>
`;
